src/Symfony/Component/Process/PhpProcess.php

```diff
@@ -39,15 +39,17 @@ public function __construct(string $script, string $cwd = null, array $env = nul
             $php = $executableFinder->find(false);
             $php = false === $php ? null : array_merge([$php], $executableFinder->findArguments());
         }
-        if ('phpdbg' === \PHP_SAPI) {
-            $file = tempnam(sys_get_temp_dir(), 'dbg');
-            file_put_contents($file, $script);
-            register_shutdown_function('unlink', $file);
-            $php[] = $file;
-            $script = null;
+
+        $file = tempnam(sys_get_temp_dir(), 'phpdbg' === \PHP_SAPI ? 'dbg' : 'cli');
+        if (false === $file) {
+            throw new RuntimeException('The system temp directory could not be written.');
```

Comment:
`... could not be written` -> `... is not writable`?

Reply:
That's better, thanks! :slightly_smiling_face: